release/util.bzl

```diff
@@ -73,6 +73,7 @@ def sdk_tarball(name, version):
           cp $(location //daml-assistant/daml-sdk:sdk_deploy.jar) $$OUT/daml-sdk/daml-sdk.jar
           cp -L $(location //ledger-service/http-json:src/main/resources/logback.xml) $$OUT/daml-sdk/json-api-logback.xml
           cp -L $(location //triggers/service:release/trigger-service-logback.xml) $$OUT/daml-sdk/
+          cp -L $(location //triggers/service/auth:release/oauth2-middleware-logback.xml) $$OUT/daml-sdk/
```

Comment:
why did we not get a test failure despite this?

Reply:
Passing a non-existing path to the `--logback-config` flag on `daml-helper run-jar` seems to be silently ignored.